src/extensions/speech_extension/commands/transcribe_command.cs

```diff
@@ -0,0 +1,323 @@
+using System.Text;
+using System.Text.Json;
+using System.Net;
+using System.Net.Http.Headers;
+
+namespace Azure.AI.Details.Common.CLI
+{
+    public class TranscribeCommand : Command
+    {
+        public TranscribeCommand(ICommandValues values) : base(values)
+        {
+            _quiet = _values.GetOrDefault("x.quiet", false);
+            _verbose = _values.GetOrDefault("x.verbose", false);
+        }
+
+        public bool RunCommand()
+        {
+            try
+            {
+                Transcribe();
+            }
+            catch (WebException ex)
+            {
+                FileHelpers.LogException(_values, ex);
+                ConsoleHelpers.WriteLineError($"\n  ERROR: {ex.Message}");
+                JsonHelpers.PrintJson(HttpHelpers.ReadWriteJson(ex.Response, _values, "transcribe"));
+            }
+
+            return _values.GetOrDefault("passed", true);
+        }
+
+        private void Transcribe()
+        {
+            StartCommand();
+
+            var content = new MultipartFormDataContent();
+            content.Add(new ByteArrayContent(FileHelpers.ReadAllBytes(GetAudioFile())), "audio", Path.GetFileName(GetAudioFile()));
+            content.Add(new StringContent(GetDefinitionJson()), "definition");
+
+            var request = CreateRequestMessage();
+            request.Content = content;
+
+            CheckWriteOutputRequest(request, content);
+
+            var client = new HttpClient();
+            var response = client.SendAsync(request).Result;
+            var json = ReadWritePrintJson(response);
+            ProcessResponse(json);
+
+            StopCommand();
+            DeleteTemporaryFiles();
+        }
+
+        private HttpRequestMessage CreateRequestMessage()
+         {
+            var region = _values["service.config.region"];
+            var url = $"https://{region}.api.cognitive.microsoft.com/speechtotext/transcriptions:transcribe?api-version=2024-11-15";
+
+            var request = new HttpRequestMessage(HttpMethod.Post, url);
+            request.Headers.Add("Ocp-Apim-Subscription-Key", GetSubscriptionKey());
```

Comment:
>request.Headers.Add("Ocp-Apim-Subscription-Key", GetSubscriptionKey()); [](http://example.com/codeflow?start=12&length=71)

Does Fast support other auth schemes?

Reply:
[Microsoft Docs](https://www.google.com/search?q=fast+transcription+site%3Amicrosoft.com) don't show any examples of anything except key based... 
[Searching](https://www.google.com/search?q=%22speechtotext/transcriptions:transcribe%22) for other examples off-site found no examples either...